docs/user_guide/configuring.rst

```diff
@@ -179,26 +179,48 @@ By default, this theme's sidebar has these two elements in it. If you'd like to
 override this behavior and control the sidebar on a per-page basis, use the
 `Sphinx html-sidebars configuration value <https://www.sphinx-doc.org/en/master/usage/configuration.html?highlight=html_sidebars#confval-html_sidebars>`_.
 
+.. navigation-depth:
 
-Configure the navigation depth and collapsing of the sidebar
-============================================================
+Navigation depth and collapsing of the sidebar
+==============================================
 
 By default, this theme enables to expand/collapse subsections in the left
 sidebar navigation (without actually navigating to the page itself), and this extends
 up to 4 levels deep:
 
 .. image:: /_static/demo-expandable-navigation.gif
 
-When having a site with many files and/or many levels, this can cause a long
-build time and larger HTML file sizes. Therefore, it is possible to turn off
-the expandable navigation by setting the `collapse_navigation` config option
-to True:
+Sites that have a lot of pages (such as API documentation with a lot of items)
+take much longer to build and will have large output sizes
+because of all the toctree links.
 
-.. code:: python
+Here are a few suggestions if you run into this problem, ordered from least-to-most drastic:
 
-   html_theme_options = {
-     "collapse_navigation": True
-   }
+.. _remove_toctrees:
+
+Selectively remove pages from your sidebar
+------------------------------------------
+
+You can prevent pages from showing up in the navigation bar using a Sphinx
+extension called [`sphinx-remove-toctrees`](https://github.com/executablebooks/sphinx-remove-toctrees).
```

Comment:
Unfortunately we are in a rst file, not md .. :) 
(something we could also change to myst of course)